packages/builder/src/components/common/DashCard.svelte

```diff
@@ -0,0 +1,63 @@
+<script>
+  import { Icon, Detail } from "@budibase/bbui"
+
+  export let title = ""
+  export let actionIcon
+  export let action
+  export let dataCy
+
+  $: actionDefined = typeof action === "function"
+</script>
+
+<div class="dash-card" data-cy={dataCy}>
+  <div
+    class={actionDefined ? "dash-card-header active" : "dash-card-header"}
```

Comment:
NAB: this is equivalent to just
```js
<div
  class="dash-card-header"
  class:active={actionDefined}
  ...
>
```
Just avoids repetition of class names.

Reply:
Updated this to follow the convention.